Makefile

```diff
@@ -1,4 +1,4 @@
-  
+
```

Comment:
I see a (final blank line)[https://github.com/appcelerator/amp/blob/master/.editorconfig#L9] specified in .editorconfig, but not for one at the top of a file...?  (As an aside, there are pros and cons for a final blank line, and technically it historically has been [required](https://robots.thoughtbot.com/no-newline-at-end-of-file#history-lesson); it shouldn't matter if it gets added to the VERSION file since at I ensure that the contents are trimmed before printing.)